_includes/sponsors_footer.html

```diff
@@ -1,4 +1,4 @@
-<h3 id="sponsors">Sponsors</h3>
+<h3>Sponsors</h3>
```

Comment:
You're right! I think we never really feel the need to do so, since there is a separate page for sponsors. Should we somehow need this again, we can think of something else :) Thank you for the review! ✨